src/classes/PS_CommitmentRequest.cls

```diff
@@ -296,6 +297,11 @@ public inherited sharing class PS_CommitmentRequest {
         public String currencyCode;
         public List<Schedule> schedules;
 
+        /***	
+         * @description Optional Metadata containing origin type and other properties	
```

Comment:
I think it would be better to either enumerate all properties or just refer to Metadata properties.